jps-plugin/src/com/goide/jps/builder/GoBuilder.java

```diff
@@ -63,7 +66,14 @@ public void build(@NotNull GoTarget target,
     JpsTypedModule<JpsSimpleElement<JpsGoModuleProperties>> module = jpsModule.asTyped(JpsGoModuleType.INSTANCE);
     assert module != null;
     JpsSdk<JpsDummyElement> sdk = getSdk(context, module);
-    File executable = GoEnvironmentUtil.getExecutableForSdk(sdk.getHomePath());
+    File exec = GoEnvironmentUtil.getExecutableForSdk(sdk.getHomePath());
+    if (exec == null) {
+      Notifications.Bus.notify(
```

Comment:
I know that the notification itself is a duplicate. Maybe it could be extracted into a common static method that could receive just the title of the notification, as that's different between them.
However, this way we can cover situations like having an update change the location of the go tool (if the user installs a new version from a package manager not from the official release) and IDEA is still running.

Should I extract the method to ```utils/src/com/goide/GoEnvironmentUtil.java``` or is there a better place for it?